tests/api/v1/aspects/get.js

```diff
@@ -80,7 +81,88 @@ describe(`api: GET ${path}`, () => {
   after(u.forceDelete);
   after(tu.forceDeleteUser);
 
+  describe('duplicate tags fail', () => {
+    it('EXCLUDE filter', (done) => {
+      api.get(`${path}?tags=-Foo,-Foo`)
+      .set('Authorization', token)
+      .expect(constants.httpStatus.BAD_REQUEST)
+      .end((err, res) => {
+        if (err) {
+          done(err);
+        }
+
+        expect(res.body.errors[0].type).to.equal('DuplicateFieldError');
+        done();
+      });
+    });
+
+    it('EXCLUDE filter case-sensitive', (done) => {
+      api.get(`${path}?tags=-Foo,foo`)
+      .set('Authorization', token)
+      .expect(constants.httpStatus.BAD_REQUEST)
+      .end((err, res) => {
+        if (err) {
+          done(err);
+        }
+
+        expect(res.body.errors[0].type).to.equal('DuplicateFieldError');
+        done();
+      });
+    });
+
+    it('INCLUDE filter', (done) => {
+      api.get(`${path}?tags=Foo,Foo`)
+      .set('Authorization', token)
+      .expect(constants.httpStatus.BAD_REQUEST)
+      .end((err, res) => {
+        if (err) {
+          done(err);
+        }
+
+        expect(res.body.errors[0].type).to.equal('DuplicateFieldError');
+        done();
+      });
+    });
+
+    it('INCLUDE filter case-sensitive', (done) => {
+      api.get(`${path}?tags=Foo,foo`)
+      .set('Authorization', token)
+      .expect(constants.httpStatus.BAD_REQUEST)
+      .end((err, res) => {
+        if (err) {
+          done(err);
+        }
+
+        expect(res.body.errors[0].type).to.equal('DuplicateFieldError');
+        done();
+      });
+    });
+  });
+
   describe('Single Values: ', () => {
+    it('filter by EXCLUDE tags returns expected values', (done) => {
```

Comment:
do we need these tests for multiple values as well? And combination of include,exclude